app/controllers/posts_controller.rb

```diff
@@ -2,6 +2,10 @@ class PostsController < ApplicationController
   def index
     @user = User.find_by(id: params[:user_id])
     @posts = @user.posts.includes(:comments).paginate(page: params[:page], per_page: 2)
+    respond_to do |format|
+      format.html # index.html.erb
+      format.json { render json: @posts }
+    end
   end
```

Comment:
- Greate this works as expected. :heavy_check_mark: 

### Result
![posts-for-a-user](https://user-images.githubusercontent.com/26597432/232092031-86a346a2-615d-4911-8bdf-52f6e82ddabb.png)